geo-types/src/geometry/multi_point.rs

```diff
@@ -235,53 +235,50 @@ mod test {
     fn test_relative_eq() {
         let delta = 1e-6;
 
-        let multi = MultiPoint::new(vec![point![x: 0., y: 0.], point![x: 10., y: 10.]]);
+        let multi = wkt! { MULTIPOINT(0. 0.,10. 10.) };
+
+        let mut multi_x = multi.clone();
+        *multi_x.0[0].x_mut() += delta;
 
-        let multi_x = MultiPoint::new(vec![point![x: 0., y: 0.], point![x: 10.+delta, y: 10.]]);
         assert!(multi.relative_eq(&multi_x, 1e-2, 1e-2));
         assert!(multi.relative_ne(&multi_x, 1e-12, 1e-12));
 
-        let multi_y = MultiPoint::new(vec![point![x: 0., y: 0.], point![x: 10., y: 10.+delta]]);
+        let mut multi_y = multi.clone();
+        *multi_y.0[0].y_mut() += delta;
         assert!(multi.relative_eq(&multi_y, 1e-2, 1e-2));
         assert!(multi.relative_ne(&multi_y, 1e-12, 1e-12));
 
         // Under-sized but otherwise equal.
-        let multi_undersized = MultiPoint::new(vec![point![x: 0., y: 0.]]);
+        let multi_undersized = wkt! { MULTIPOINT(0. 0.) };
         assert!(multi.relative_ne(&multi_undersized, 1., 1.));
 
         // Over-sized but otherwise equal.
-        let multi_oversized = MultiPoint::new(vec![
-            point![x: 0., y: 0.],
-            point![x: 10., y: 10.],
-            point![x: 10., y:100.],
-        ]);
+        let multi_oversized = wkt! { MULTIPOINT(0. 0.,10. 10.,10. 100.) };
         assert!(multi.relative_ne(&multi_oversized, 1., 1.));
     }
 
     #[test]
     fn test_abs_diff_eq() {
         let delta = 1e-6;
 
-        let multi = MultiPoint::new(vec![point![x: 0., y: 0.], point![x: 10., y: 10.]]);
+        let multi = wkt! { MULTIPOINT(0. 0.,10. 10.) };
 
-        let multi_x = MultiPoint::new(vec![point![x: 0., y: 0.], point![x: 10.+delta, y: 10.]]);
+        let mut multi_x = multi.clone();
+        *multi_x.0[0].x_mut() += delta;
```

Comment:
Here's an example of where it's less nice. Unlike the `serde_json::json!` macro, you can't embed expressions, the `wkt!` macro only accepts literals.